README.md

```diff
@@ -1,9 +1,11 @@
 # Help desk
 
-Support for Infrastructure related issues for the Jenkins project
+Support for Infrastructure related issues for the Jenkins project.
 
-As of the 11th of January 2022, all existing Jira INFRA project have been imported in this repository as Github issues.
-See https://github.com/jenkins-infra/helpdesk/issues/9 for more details on the migration. 
+If you want to report an issue or a bug with the Jenkins website, wiki, or any other [services run by the Jenkins project](https://www.jenkins.io/projects/infrastructure/), [open an issue here](https://github.com/jenkins-infra/helpdesk/issues/new/choose)!
```

Comment:
this looks like it was copied from an old version of the website PR?

We removed wiki and website from the text there as neither of them made sense to have an issue opened here?

Reply:
Yeah, there is old comment here, probably my mistake, taking back the version suggested by Mark in the previous PR.

Reply:
It was on another PR: https://github.com/jenkins-infra/jenkins.io/pull/4816#discussion_r783214973